src/hooks.ts

```diff
@@ -73,3 +73,17 @@ function convertValueAbs(
   if (scale === undefined) return 0;
   return typeof value === 'number' ? Math.abs(scale(0) - scale(value)) : value;
 }
+
+export function usePointPosition(config: UsePositionConfig[]) {
+  const { axisContext } = usePlotContext();
+  const [xScale, yScale] = validateAxis(axisContext, 'x', 'y');
+  const points = config;
+  let result = '';
+  points.forEach((point) => {
```

Comment:
I didn't benchmark this code but I don't think it is the best way to do it because you create huge amount of intermediate strings.

Use rather something like:
```
return points.map((point) => `${convertValue(point.x, xScale)},${convertValue(
      point.y,
      yScale,
    )`).join(' ')
```

Reply:
yes better